### PR TITLE
Move original kep from design proposals to new home and update

### PR DESCRIPTION
**Pull request by timothysc**
This PR moves one of the original piloted keps into it's proper home and updates to reflect changes in the spec.

### CODE REVIEW

File: keps/sig-cluster-lifecycle/0004-bootstrap-checkpointing.md [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+status: implemented
 authors:
-  - name: Timothy St. Clair
-    github: @timothysc
-    email: timothysc@gmail.com
+  - "@timothysc"
 owning-sig: sig-cluster-lifecycle 
-participating-sigs:
-  - sig-node
+participating-sigs: sig-node
```
Comment (roberthbailey) [inline keps/sig-cluster-lifecycle/0004-bootstrap-checkpointing.md]: This should be in list format, e.g.

```
participating-sigs:
  - sig-node
```

Comment (timothysc) [reply keps/sig-cluster-lifecycle/0004-bootstrap-checkpointing.md]: I don't know... @jbeda?

Comment (jbeda) [reply keps/sig-cluster-lifecycle/0004-bootstrap-checkpointing.md]: Yeah -- these should be a list.  We should create a presubmit check that validates the schema in front of each KEP.

**Comment by timothysc** [comment]
comments addressed - PTAL.

**Comment by timothysc** [comment]
/lgtm 
self-applying

**Pull request merged**